core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/metrics/TableRepairMetrics.java

```diff
@@ -40,6 +40,13 @@
      */
     void lastRepairedAt(TableReference tableReference, long lastRepairedAt);
 
+    /**
+     * Report the remaining repair time for table.
+     * @param tableReference The table to update the remaining repair time for.
+     * @param remainingRepairTime The remaining time to fully repair the table.
```

Comment:
Probably good to mention that this is active repair time.
Aka the time spend waiting for cassandra to repair as opposed to the time for a job to complete.